report.tex

```diff
@@ -259,7 +259,7 @@ \section*{Appendices}
 \section{Notes on the Current UI}
 We found a number of significant problems with the current UI, leading us to suggest a complete revamp of the frontend of the application. The first of which is the navigation pain, as discussed in section \ref{remediations}, but particularly the lack of useful information provided in any given program state. One way to fix this would be to use the loading screens as a form of tutorial, teaching the user how to perform tasks in the program, but as stated above, there is too much useless interaction with the home pages.
 
-Currently, the program also fails to offer a `dashboard-esque' area for clients to view their created scenes or get a feel for the global structure of their scenario. We did not identify an easy way to solve this problem in the current UI paradigm, despite the implementation of a preview pane. Additionally there are misleading interactions in the current UI caused by a clash between the classical implementation of the tab-bar and the implementation presented in the \textit{Scenario Builder} such as the arrow buttons (which we initially expect to move us between tabs) and the drop-down menu for each tab (whose purpose we fail to understand).
+Currently, the program also fails to offer a `dashboard-esque' area for clients to view their created scenes or get a feel for the global structure of their scenario. We did not identify an easy way to solve this problem in the current UI paradigm, despite the implementation of a preview pane. Additionally, there are misleading interactions in the current UI caused by a clash between the classical implementation of the tab-bar and the implementation presented in the \textit{Scenario Builder}, such as the arrow buttons (which we initially expect to move us between tabs) and the drop-down-delete menu for each tab (whose purpose and execution is flawed).
```

Comment:
maybe we just change it to 'the drop-down-delete menu that could be moved into the tab itself.'